subsys/bluetooth/controller/ll_sw/nordic/lll/lll_tmp.c

```diff
@@ -0,0 +1,240 @@
+#include <stdbool.h>
+#include <stddef.h>
+
+#include <toolchain.h>
+#include <zephyr/types.h>
+
+#if defined(CONFIG_BT_CTLR_DEBUG_PINS)
+#if defined(CONFIG_PRINTK)
+#undef CONFIG_PRINTK
+#endif
+#endif
+
+#include "hal/ccm.h"
+
+#include "util/mfifo.h"
+#include "util/memq.h"
+
+#include "ticker/ticker.h"
+
+#include "pdu.h"
+
+#include "lll.h"
+#include "lll_conn.h"
+#include "lll_tmp.h"
+#include "lll_internal.h"
+
+#include "common/log.h"
+#include <soc.h>
+#include "hal/debug.h"
+
+static MFIFO_DEFINE(tmp_ack, sizeof(struct lll_tx),
+		    CONFIG_BT_TMP_TX_COUNT_MAX);
+
+static int _init_reset(void);
+static int _prepare_cb(struct lll_prepare_param *prepare_param);
+static int _is_abort_cb(void *next, int prio, void *curr,
+			lll_prepare_cb_t *resume_cb, int *resume_prio);
+static void _abort_cb(struct lll_prepare_param *prepare_param, void *param);
+static int _emulate_tx_rx(void *param);
+
+int lll_tmp_init(void)
+{
+	int err;
+
+	err = _init_reset();
+	if (err) {
+		return err;
+	}
+
+	return 0;
+}
+
+int lll_tmp_reset(void)
+{
+	int err;
+
+	MFIFO_INIT(tmp_ack);
+
+	err = _init_reset();
+	if (err) {
+		return err;
+	}
+
+	return 0;
+}
+
+void lll_tmp_prepare(void *param)
+{
+	struct lll_prepare_param *p = param;
+	int err;
+
+	printk("\t\tlll_tmp_prepare (%p) enter.\n", p->param);
```

Comment:
Are those `printk` staatements intentional?

Reply:
tmp was  a template role, this I will remove as part of the rework, this should not be part of the merge into master.